src/test/lambda/wizards/uploadLambdaWizard.test.ts

```diff
@@ -4,9 +4,11 @@
  */
 
 import * as vscode from 'vscode'
+import { writeFileSync } from 'fs-extra'
 import { createWizardTester, WizardTester } from '../../shared/wizards/wizardTestUtils'
 import { UploadLambdaWizard, UploadLambdaWizardState, LambdaFunction } from '../../../lambda/commands/uploadLambda'
 import { makeTemporaryToolkitFolder } from '../../../shared/filesystemUtilities'
+import path = require('path')
```

Comment:
I think we generally use `*` imports instead of require. E.g. `import * as path from 'path'`

Reply:
Good call, it was auto imported and I missed that.